app/src/main/java/ru/vssemikoz/newsfeed/api/NewsApi.java

```diff
@@ -8,10 +8,14 @@
 public interface NewsApi {
     @GET("/v2/top-headlines")
     Call<NewsApiResponse> getNewsByCategory(@Query("country") String country,
-                                  @Query("apiKey") String key,
-                                  @Query("category") String category);
+                                            @Query("apiKey") String key,
+                                            @Query("category") String category);
 
     @GET("/v2/top-headlines")
-    Call<NewsApiResponse> getNews(@Query("country") String country,
-                                            @Query("apiKey") String key);
+    Call<NewsApiResponse> getTopNews(@Query("country") String country,
+                                     @Query("apiKey") String key);
+
+    @GET("/v2/top-headlines")
+    Call<NewsApiResponse> getNewsBySource(@Query("sources") String sources,
```

Comment:
There should be only 1 method which would be used with categories/sources and without them at all. AFAIK, passing null as the parameter will omit it from the request so if you will have `ALL` in the sources or in the channels you can just pass null there and receive a list that wouldn't be filtered by those params.